prowler/providers/aws/services/dlm/dlm_ebs_snapshot_policy_exists/dlm_ebs_snapshot_policy_exists.py

```diff
@@ -0,0 +1,15 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.dlm.dlm_client import dlm_client
+
+
+class dlm_ebs_snapshot_policy_exists(Check):
+    def execute(self):
+        findings = []
+        report = Check_Report_AWS(self.metadata())
+        report.status = "FAIL"
+        report.status_extended = "No EBS snapshot policies found."
```

Comment:
This rule is verifying if at least one EBS snapshots lifecycle exists into the AWS account, is not checking if every volume are at least covered by one policy. Considering the check is doing makes no sense to include the data you requested in the report. Maybe can make sense to change the rule to link every volume to the snapshot lifecycle polices by it's tags, and in that case include the volume arn, id, region, and tags in the results.

Reply:
Hi @jit-contrib, regardless the purpose of the check, all checks must fill the `resource_arn`, `resource_id`, `region` and `tags` (if possible). Please refer to the Developer Guide here https://docs.prowler.cloud/en/latest/developer-guide/checks/#resource-id-name-and-arn because if the resource audited is the AWS account these fields will be as follows:
- `resource_arn` --> AWS Account Root ARN, like `arn:aws:iam::123456789012:root`
- `resource_id` --> AWS Account number, like `123456789012`
- `region` --> The `service.region` object.


Please, review the developer guide and give us feedback is something is not present or clear.

Thanks!